ui/src/submissions/common/components/RichTextEditor/RichTextEditor.jsx

```diff
@@ -1,10 +1,15 @@
 import React, { Component } from 'react';
-import QuillEditor from 'react-quill';
+import QuillEditor, { Quill } from 'react-quill';
 
 import 'react-quill/dist/quill.snow.css';
 import './RichTextEditor.scss';
 import EditorToolbar from './EditorToolbar';
 
+// change default text default (`P`)
+const Block = Quill.import('blots/block');
+Block.tagName = 'DIV';
```

Comment:
that's not so nice as tt's really a paragraph semantically :disappointed: can we strip empty paragraphs instead? or change the style in Quill?

Reply:
It's already in the commit message, If we change the styles for Quill, then we lose ability to do linebreaks without blank space, as it would create `p` whenever you hit enter, and each `p` would have a `margin-bottom: 1em`